server/game/core/Game.js

```diff
@@ -903,34 +907,34 @@ class Game extends EventEmitter {
     /**
      * Creates a "sub-window" for events which will have priority resolution and
      * be resolved immediately after the currently resolving set of events, preceding
-     * the next steps of any ability being triggered.
+     * the next steps of any ability being resolved.
      *
      * Typically used for defeat events.
      */
     addSubwindowEvents(events) {
         this.currentEventWindow.addSubwindowEvents(events);
     }
 
-    /**
-     * Raises a custom event window for checking for any cancels to a card
-     * ability
-     * @param {Object} params
-     * @param {Function} handler - this is an arrow function which is called if
-     * nothing cancels the event
-     */
-    raiseInitiateAbilityEvent(params, handler) {
-        this.raiseMultipleInitiateAbilityEvents([{ params: params, handler: handler }]);
-    }
+    // /**
```

Comment:
Question: Should we have `TODO` comments associated with these?

Reply:
While we're in the pre-release timeframe my habit is to just comment out blocks of jinteki stuff that we aren't sure will be needed, so any large block comments like this can be safely assumed to be that. I'm not outright removing in case there's some chance we end up needing it so we can find it again later.

We'll do a cleanup pass for all of this once we get closer to release.